README.md

```diff
@@ -25,6 +25,15 @@ job lifecycle management are the responsibility of cluster-autoscaler,
 kube-scheduler and kube-controller-manager, respectively. Advanced
 admission control can be delegated to controllers such as [gatekeeper](https://github.com/open-policy-agent/gatekeeper).
 
+## Production Readiness status
+
+- ✔️ Coverage with unit and integration tests: ~72%
+- ✔️ Up-to-date [documentation](/docs).
+- ✔️ Monitoring via [metrics](/docs/reference/metrics.md)
+- ✔️ API validation and defaulting.
+- ⏳ Scalability tests in progress.
+- ⚠️ Latest API version: v1alpha2.
```

Comment:
Do we want to add a note below about this item and why it is in this state? Something along the lines of: based on community feedback we continue to evolve the API to address new use cases and make it more intuitive to use.

Reply:
Done